build.gradle.kts

```diff
@@ -1,226 +1,193 @@
-import com.adarshr.gradle.testlogger.theme.ThemeType
-import org.jetbrains.intellij.tasks.RunPluginVerifierTask.VerificationReportsFormats.*
-import org.jetbrains.intellij.tasks.RunPluginVerifierTask.FailureLevel.*
-
-fun properties(key: String) = providers.gradleProperty(key)
-fun environment(key: String) = providers.environmentVariable(key)
-fun prop(name: String): String {
-    return properties(name).get()
-}
+import org.jetbrains.changelog.Changelog
+import org.jetbrains.changelog.markdownToHTML
+import org.jetbrains.intellij.platform.gradle.TestFrameworkType
+
 plugins {
-    java // Java support
+    id("java") // Java support
     alias(libs.plugins.kotlin) // Kotlin support
-    alias(libs.plugins.gradleIntelliJPlugin) // Gradle IntelliJ Plugin
+    alias(libs.plugins.intelliJPlatform) // IntelliJ Platform Gradle Plugin
     alias(libs.plugins.changelog) // Gradle Changelog Plugin
-    alias(libs.plugins.testLogger) // Nice test logs
```

Comment:
please keep testlogger plugin to provide nice tests output ( diffcult to read without it ) .
also, remove qodana as it's not used by our extensio nto review idea inspections.